fbw-a380x/src/wasm/fbw_a380/src/interface/SimConnectInterface.cpp

```diff
@@ -2319,6 +2319,9 @@ void SimConnectInterface::processEventWithOneParam(const DWORD eventId, const DW
     case Events::AUTO_THROTTLE_DISCONNECT: {
       simInputThrottles.ATHR_disconnect = 1;
       std::cout << "WASM: event triggered: AUTO_THROTTLE_DISCONNECT" << std::endl;
+
+      // Re emitting masked event for autobrake disconnection
+      execute_calculator_code("(>K:A32NX.AUTO_THROTTLE_DISCONNECT)", nullptr, nullptr, nullptr);
```

Comment:
```suggestion
      execute_calculator_code("(>K:A32NX_AUTO_THROTTLE_DISCONNECT)", nullptr, nullptr, nullptr);
```

Reply:
Periods are required in custom event names according to the MSFS SDK docs

Reply:
Didn't know that. Thanks for the heads up.

Reply:
This looks strange - why would the AUTO_THROTTLE_DISCONNECT itself re-emmit itself - wouldn't that cause a loop?